setup.py

```diff
@@ -1,7 +1,8 @@
 from setuptools import setup
 
 dependencies = [
-    'redis==3.5.3',
+    'redis==4.5.4;python_version >= "3.0"',
```

Comment:
```suggestion
    'redis>=4.5.4;python_version >= "3.0"',
```
The true versions are specified by Dockerfile, such as dockers/docker-config-engine-bullseye/Dockerfile.j2. It may use later versions, we do not want setup.py to install an older version in that case.

Reply:
It is a good suggestion, I will send a PR to fix the master branch.